receiver/mongodbreceiver/scraper.go

```diff
@@ -202,3 +213,23 @@ func (s *mongodbScraper) recordAdminStats(now pcommon.Timestamp, document bson.M
 func (s *mongodbScraper) recordIndexStats(now pcommon.Timestamp, indexStats []bson.M, databaseName string, collectionName string, errs *scrapererror.ScrapeErrors) {
 	s.recordIndexAccess(now, indexStats, databaseName, collectionName, errs)
 }
+
+func serverAddressAndPort(serverStatus bson.M) (string, int64, error) {
```

Comment:
Would it be helpful to have named return arguments here? Would it be more clear if we named the method `getServerAddressAndPort`? This is pretty simple method, so neither of these things are that big of a deal, just slightly more clear on purpose and functionality.

_Just some nits, feel free to drop if they're not helpful._

Reply:
> Would it be helpful to have named return arguments here?

I think the name of the function server**Address**And**Port** plays the role of named return arguments here

> Would it be more clear if we named the method getServerAddressAndPort? 

get prefix is an anti-pattern in go :) https://go.dev/doc/effective_go#Getters

Reply:
Fair point on the return arguments clarity. Thanks for sharing the docs reference too, I'm definitely wrong there 😅